src/assets/sass/utils/_sizing.scss

```diff
@@ -1,58 +1,85 @@
-.w60 {
-  width: 60% !important;
+.wAuto {
+  width: auto !important;
 }
-.h60 {
-  height: 60% !important;
+@for $value from 0 through 100 {
+  .w#{$value} {
+    width: percentage($value/100) !important;
+  }
 }
-.w80 {
-  width: 80% !important;
+@for $value from 0 through 300 {
```

Comment:
@farmanahmed2007 100vw is maximum or what is 300 here?

Reply:
make it 100